OracleWebLogic/samples/wls-k8s-domain/Dockerfile

```diff
@@ -0,0 +1,36 @@
+
+# Pull base image
+# ---------------
+FROM weblogic-12.2.1.3-developer:latest
+
+MAINTAINER Lily He <lily.he@oracle.com>
+
+# Environment variables required for this build (do NOT change)
+# -------------------------------------------------------------
+ENV MW_HOME="$ORACLE_HOME" \
+    PATH="$ORACLE_HOME/wlserver/server/bin:$ORACLE_HOME/wlserver/../oracle_common/modules/org.apache.ant_1.9.2/bin:$JAVA_HOME/jre/bin:$JAVA_HOME/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$ORACLE_HOME/oracle_common/common/bin:$ORACLE_HOME/wlserver/common/bin:$ORACLE_HOME/wlserver/samples/server/" \
+    WLST="$ORACLE_HOME/oracle_common/common/bin/wlst.sh" \
+    DOMAIN_NAME=wlsdomain \
+    SAMPLE_DOMAIN_HOME=/u01/wlsdomain \
+    ADMIN_PORT=8001
+
+user root
+# Copy scripts and install python http lib
+# --------------------------------
+COPY container-scripts/ /u01/oracle/
+RUN chmod +x /u01/oracle/*.sh /u01/oracle/*.py
+
+# The following installation need internet access. If you are behind a proxy you may need to set http_proxy and https_proxy
```

Comment:
Please remove this comment and the following two lines (23-25) as proxies should be provided as a built-time argument (`--build-arg`) and not be specified in the `Dockerfile`.